### PR TITLE
EMKTG-3393 - Field requiered in Attribute Mapping

### PR DESCRIPTION
**Pull request by ClaraLpresta**

### CODE REVIEW

File: _dev/apps/ui/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue [hunk 1]
```diff
@@ -280,6 +281,10 @@ export default defineComponent({
       return this.$t(`tooltip.attributeMapping.${name}`);
     },
     nextStep() {
+      this.mappingAttributes.forEach((attribute) => {
+        this.hasError = attribute.fields.some((value) => value.mapped?.length === 0);
+      });
+      if (this.hasError) return;
```
Comment (ga-devfront) [inline _dev/apps/ui/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue:287]: It's not good. We will test all the values ​​one by one and directly change the hasError however if the last one does not have one the field will be false THEN there may be errors before.
You should rather reset the hasError at the beginning (false) then do a hasError = myMethod || hasError to let the possibility to be true

**Review by ga-devfront — CHANGES_REQUESTED** [review]

**Review by ga-devfront — COMMENTED** [review]
another thing. Why do you use a global variable if you don't use it anywere else ? You can just use a const in this case no ?

**Review by ga-devfront — APPROVED** [review]
the difference is not huge, but when declared in data() the data persists and therefore requires more resources. Except here, we don't need its persistence, because we recalculate its value each time without it impacting the rest.

**Comment by ClaraLpresta** [comment]
@ga-devfront I understand, good to know! Thanks

**Pull request merged**